sw/device/silicon_creator/mask_rom/BUILD

```diff
@@ -0,0 +1,253 @@
+        "//sw/device/silicon_creator/lib:irq_asm",
+        "//sw/device/silicon_creator/lib/base:abs_mmio",
+        "//sw/device/silicon_creator/lib/base:sec_mmio",
+        "//sw/device/silicon_creator/lib/base:static_critical.ld",
+        "//sw/device/silicon_creator/lib/drivers:flash_ctrl",
+        "//sw/device/silicon_creator/lib/drivers:uart",
+    ],
+)
+
+cc_library(
+    name = "primitive_bootstrap",
+    srcs = ["primitive_bootstrap.c"],
+    hdrs = ["primitive_bootstrap.h"],
+    deps = [
+        "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+        "//sw/device/boot_rom:bootstrap",
+        "//sw/device/lib:flash_ctrl",
+        "//sw/device/lib/arch:device",
+        "//sw/device/lib/base",
+        "//sw/device/lib/dif:gpio",
+        "//sw/device/lib/dif:spi_device",
+        "//sw/device/silicon_creator/lib:error",
+        "//sw/device/silicon_creator/lib:log",
+        "//sw/device/silicon_creator/lib/drivers:hmac",
+    ],
+)
+
+cc_library(
+    name = "sigverify_keys_ptrs",
+    hdrs = ["sigverify_keys_ptrs.h"],
+)
+
+cc_library(
+    name = "mock_sigverify_keys_ptrs",
+    testonly = True,
+    hdrs = ["mock_sigverify_keys_ptrs.h"],
+    deps = [
+        ":sigverify_keys_ptrs",
+        "//sw/device/silicon_creator/testing:mask_rom_test",
+        "@googletest//:gtest",
+    ],
+)
+
+cc_library(
+    name = "sigverify_keys",
+    srcs = ["sigverify_keys.c"],
+    hdrs = ["sigverify_keys.h"],
+    deps = [
+        ":sigverify_keys_ptrs",
+        "//hw/ip/otp_ctrl/data:otp_ctrl_regs",
+        "//sw/device/lib/base",
+        "//sw/device/silicon_creator/lib:error",
+        "//sw/device/silicon_creator/lib:sigverify_internal",
+        "//sw/device/silicon_creator/lib:sigverify_intf",
+        "//sw/device/silicon_creator/lib/drivers:lifecycle",
+        "//sw/device/silicon_creator/lib/drivers:otp",
+        "//sw/device/silicon_creator/mask_rom/keys:test_keys",
+    ],
+)
+
+cc_test(
+    name = "sigverify_keys_unittest",
+    srcs = [
+        "sigverify_keys.c",
+        "sigverify_keys.h",
+        "sigverify_keys_unittest.cc",
+    ],
+    defines = ["OT_OFF_TARGET_TEST"],
+    deps = [
+        ":mock_sigverify_keys_ptrs",
+        ":sigverify_keys",
+        ":sigverify_keys_ptrs",
+        "//hw/ip/otp_ctrl/data:otp_ctrl_regs",
+        "//sw/device/lib/base",
+        "//sw/device/silicon_creator/lib:error",
+        "//sw/device/silicon_creator/lib:mock_sigverify",
+        "//sw/device/silicon_creator/lib:sigverify",
+        "//sw/device/silicon_creator/lib:sigverify_internal",
+        "//sw/device/silicon_creator/lib:sigverify_intf",
+        "//sw/device/silicon_creator/lib:sigverify_mod_exp_ibex",
+        "//sw/device/silicon_creator/lib/drivers:mock_lifecycle",
+        "//sw/device/silicon_creator/lib/drivers:mock_otp",
+        "//sw/device/silicon_creator/testing:mask_rom_test",
+        "@googletest//:gtest_main",
+    ],
+)
+
+opentitan_functest(
+    name = "mask_rom_test",
+    srcs = [
```

Comment:
There's a bunch of places where we're inconsistent about single-item lists having newlines in them like this or not... I'm surprised buildozer/buildifier doesn't do something consistent... am I missing something?

Reply:
I'm also surprised by that.  I'll see if I can figure out why.